docs/operate/get-started/other-hardware/_index.md

```diff
@@ -559,26 +559,63 @@ Make sure to physically connect your sensor to your machine's computer to prepar
 **1. Prepare to run your module**
 
 {{< tabs >}}
-{{% tab name="Python: pyinstaller (recommended)" %}}
-{{% alert title="Note" color="note" %}}
-To follow these PyInstaller packaging steps, you must have enabled cloud build when moving through the module generator prompts.
-If you did not, you will need to manually create a <file>build.sh</file> entrypoint script.
-{{% /alert %}}
+{{% tab name="Python: PyInstaller (recommended)" %}}
```

Comment:
I think this should be named `Python: Hot Reloading (recommended)`. PyInstaller is used later in the docs to package the module for the registry.

Reply:
Yeah... I named all the "recommended" steps the same way so that persistent tabs would align them all but maybe worth it to have a better label here
```suggestion
{{% tab name="Python: Hot reloading (recommended)" %}}
```